packages/react/src/components/Flex/Flex.tsx

```diff
@@ -71,7 +71,7 @@ class Flex extends UIComponent<ReactProps<FlexProps>> {
 
     space: PropTypes.oneOf(['around', 'between', 'evenly']),
 
-    gap: PropTypes.oneOf(['gap.small', 'gap.medium', 'gap.large']),
+    gap: PropTypes.oneOf(['gap.small', 'gap.medium', 'gap.large', 'gap.smaller']),
```

Comment:
NIT: we can order by size:` 'gap.smaller' | 'gap.small' | 'gap.medium' | 'gap.large'`